lib/controllers/status-bar-tile-controller.js

```diff
@@ -5,6 +5,7 @@ import BranchView from '../views/branch-view';
 import BranchMenuView from '../views/branch-menu-view';
 import PushPullView from '../views/push-pull-view';
 import ChangedFilesCountView from '../views/changed-files-count-view';
+import GithubStatusBarTile from '../views/github-status-bar-tile';
```

Comment:
Here's a suggestion for the class name based on package convention (ending with the word "view") and context (being in the `status-bar-tile-controller.js` file) -- `GitHubTileView`. And the file could be `github-tile-view.js`.

Similarly, since you changed the tile for opening the Git panel it might be more appropriately named something like `GitTileView` and the file `git-tile-view.js`. 

Corresponding test files would need to be changed accordingly.

Reply:
good call!  I'll change the name.